### PR TITLE
Add Badges

### LINKED ISSUE #341: Feature Request: Website Contributor Icons/Emojis
**Is your feature request related to a problem?**
No, however this has been discussed before and I think it would be a good idea to have contributor badges of some sort on the forum helpers list page. Also possibly for dashboard admins. It could be just an emoji next to their username, and I think it would be okay if it was hardcoded instead of being updated by the dashboard.

**Describe alternatives you've considered**
Not really sure

**Additional context**
None

### PR DESCRIPTION
**Pull request by Accio1**
### Resolves:

Resolves #341 

### Changes:

- Add three possible badges to the list of Forum Helpers Page.
  - Dashboard Admin
  - Site Contributor
  - Original Studio Member
  - I decided not to add any badges related to post count, because that is something that may change fairly often and therefore could require a lot of changes. I also wanted to avoid promoting post farming/grinding. If reviewers still think these badges should be added, feel free to say so.
- The people who should get badges have their badges automatically added after the page loads.
- Add an expandable legend to the top of the page explaining what each badge is for.
- To update who gets what badge, the memberData.js file will have to be updated.
  - For now, this is a hardcoded file. In the future, it may be a dashboard feature instead.

### Local Tests:

Tested Locally, screenshots attached
On a computer
image

Mobile dimensions
image

@leahcimto @penguinmoose

### CODE REVIEW

**Review by leahcimto — APPROVED** [review]
Looks good!

**Comment by leahcimto** [comment]
Also, @Accio1 I was adding some dashboard things and was wondering if you could rename wangat to KangaCoder? Thanks

**Comment by Accio1** [comment]
> Also, @Accio1 I was adding some dashboard things and was wondering if you could rename wangat to KangaCoder? Thanks

Done. In the future if I'm not active or whatever you can achieve the same result by removing the user with the original name and then adding another user with the new name. It should be a fairly rare occurrence though because of how infrequently username changes occur on scratch.

**Comment by leahcimto** [comment]
> > Also, @Accio1 I was adding some dashboard things and was wondering if you could rename wangat to KangaCoder? Thanks
> 
> Done. In the future if I'm not active or whatever you can achieve the same result by removing the user with the original name and then adding another user with the new name. It should be a fairly rare occurrence though because of how infrequently username changes occur on scratch.

I'm pretty sure though that would move the user to the bottom of the list though?

**Comment by Accio1** [comment]
> > > Also, @Accio1 I was adding some dashboard things and was wondering if you could rename wangat to KangaCoder? Thanks
> > 
> > 
> > Done. In the future if I'm not active or whatever you can achieve the same result by removing the user with the original name and then adding another user with the new name. It should be a fairly rare occurrence though because of how infrequently username changes occur on scratch.
> 
> I'm pretty sure though that would move the user to the bottom of the list though?

Right, I hadn't considered that. It's such a rare occurrence that I don't mind just directly editing the database when it occurs since adding a username change option to the dashboard would be a bit of a hassle.

**Review by penguinmoose — APPROVED** [review]
Looks good.

File: forumhelpers/FHP.js [hunk 1]
```diff
@@ -80,6 +85,19 @@ function getCount(name) {
 		});
 }
 
+function addBadges(name) {
+	var badgeBlock = document.getElementById(name+"BadgeBlock");
+	if (memberDashboardAdmins.includes(name)) {
+		badgeBlock.innerHTML = badgeBlock.innerHTML + '<img src="../resources/admin.png" class="memberBadge" width="32px" height="32px" title="Dashboard Admin">';
+	}
+	if (memberSiteContributor.includes(name)) {
+		badgeBlock.innerHTML = badgeBlock.innerHTML + '<img src="../resources/coder.png" class="memberBadge" width="32px" height="32px" title="Site Contributer">';
```
Comment (BarelySmooth) [inline forumhelpers/FHP.js:94]: > Site Contributer

typo

Comment (leahcimto) [reply forumhelpers/FHP.js:94]: #361 created

**Pull request merged**